snippets/asn.xml

```diff
@@ -1,36 +1,30 @@
 <?xml version="1.0" encoding="utf-8"?>
 <snippets>
-<snippet group="ASN.1" trigger="sequence" id="sample_snippet_definition">Sample DEFINITIONS ::= BEGIN
-
-    $name$ ::= SEQUENCE {
-        trackingNumber INTEGER,
-        question       IA5String
-    }
-
-    Test ::= SEQUENCE {
-        other $name$
-    }
-
-END
+<snippet group="ASN.1" trigger="sequence" id="sequence">SEQUENCE
+{
+    $$
+}
+</snippet>
+<snippet group="ASN.1" trigger="enumerated" id="enumerated">ENUMERATED
```

Comment:
I wonder - should we create snippets for what is after `::=` or to include assignment in snippet. Or maybe we should come out with some convention and add snippets for "enumerated + assignment", something like "asequence"

Reply:
added 'with assignment' versions of snippets